android/expoview/build.gradle

```diff
@@ -117,68 +117,65 @@ dependencies {
+  compileOnly 'host.exp.exponent:expo-camera-interface:2.0.0'
+  compileOnly 'host.exp.exponent:expo-contacts:2.0.0'
+  compileOnly 'host.exp.exponent:expo-face-detector:2.0.0'
+  compileOnly 'host.exp.exponent:expo-font:2.0.0'
   compileOnly 'host.exp.exponent:expo-gl:2.0.0'
   compileOnly 'host.exp.exponent:expo-gl-cpp:2.0.0'
-  compileOnly 'host.exp.exponent:expo-sms:2.0.0'
+  compileOnly 'host.exp.exponent:expo-google-sign-in:2.0.0'
+  compileOnly 'host.exp.exponent:expo-image-loader-interface:2.0.0'
+  compileOnly 'host.exp.exponent:expo-image-manipulator:1.0.0'
+  compileOnly 'host.exp.exponent:expo-image-picker:1.0.0'
   compileOnly 'host.exp.exponent:expo-keep-awake:1.0.0'
   compileOnly 'host.exp.exponent:expo-linear-gradient:1.0.0'
-  compileOnly 'host.exp.exponent:expo-print:2.0.0'
-  compileOnly 'host.exp.exponent:expo-media-library:2.0.0'
-  compileOnly 'host.exp.exponent:expo-analytics-segment:2.0.0'
-  compileOnly 'host.exp.exponent:expo-font:2.0.0'
-  compileOnly 'host.exp.exponent:expo-google-sign-in:2.0.0'
-  compileOnly 'host.exp.exponent:expo-location:2.0.1'
-  compileOnly 'host.exp.exponent:expo-barcode-scanner-interface:2.0.0'
-  compileOnly 'host.exp.exponent:expo-barcode-scanner:2.0.0'
-  compileOnly 'host.exp.exponent:expo-contacts:2.0.0'
-  compileOnly 'host.exp.exponent:expo-ads-admob:2.0.0'
-  compileOnly 'host.exp.exponent:expo-ads-facebook:1.0.0'
   compileOnly 'host.exp.exponent:expo-local-authentication:2.0.0'
   compileOnly 'host.exp.exponent:expo-localization:2.0.0'
-  compileOnly 'host.exp.exponent:expo-react-native-adapter:2.0.0'
-  compileOnly 'host.exp.exponent:expo-calendar:1.0.0'
-  compileOnly 'host.exp.exponent:expo-camera:2.0.0'
-  compileOnly 'host.exp.exponent:expo-camera-interface:2.0.0'
+  compileOnly 'host.exp.exponent:expo-location:2.0.1'
+  compileOnly 'host.exp.exponent:expo-mail-composer:1.0.0'
+  compileOnly 'host.exp.exponent:expo-media-library:2.0.0'
+  compileOnly 'host.exp.exponent:expo-payments-stripe:2.0.0'
+  compileOnly 'host.exp.exponent:expo-print:2.0.0'
   compileOnly 'host.exp.exponent:expo-sensors:2.0.0'
-  compileOnly 'host.exp.exponent:expo-sensors-interface:2.0.0'
+  compileOnly 'host.exp.exponent:expo-sms:2.0.0'
   compileOnly 'host.exp.exponent:expo-speech:1.0.0'
-  compileOnly 'host.exp.exponent:expo-face-detector:2.0.0'
-  compileOnly 'host.exp.exponent:expo-payments-stripe:2.0.0'
-  compileOnly 'host.exp.exponent:expo-permissions:2.0.0'
-  compileOnly 'host.exp.exponent:expo-permissions-interface:2.0.0'
-  compileOnly 'host.exp.exponent:expo-background-fetch:1.0.0'
-  compileOnly 'host.exp.exponent:expo-mail-composer:1.0.0'
   compileOnly 'host.exp.exponent:expo-sqlite:1.0.0'
-  compileOnly 'host.exp.exponent:expo-image-picker:1.0.0'
   compileOnly 'host.exp.exponent:expo-web-browser:1.0.0'
   END UNCOMMENT WHEN DISTRIBUTING */
 
   // WHEN_DISTRIBUTING_REMOVE_FROM_HERE
   api project(':ReactAndroid')
 
   // Universal modules
-  api project(':expo-react-native-adapter')
```

Comment:
See https://github.com/expo/expo/pull/3245/files#diff-f84493f6c824ac50b22a2140cfc393f3R186

Reply:
🤔 the link outdated I think